build.gradle

```diff
@@ -46,19 +47,28 @@ repositories {
     mavenCentral()
 }
 
-
-repositories {
-    maven {
-        name = "Synology"
-
-        def baseUrl = "file://Volumes/Builds/qa/automation/maven-repository"
-        def releasesRepoUrl = "$baseUrl/releases"
-        def snapshotsRepoUrl = "$baseUrl/snapshots"
-        url = version.endsWith('SNAPSHOT') ? snapshotsRepoUrl : releasesRepoUrl
+publishing {
+    publications {
+        maven(MavenPublication) {
+            groupId = group
+            artifactId = artifact
```

Comment:
I am not a gradle expert by any means, but I'm curious why this is a variable above and then only used here?

Reply:
just to keep it close to the others

Reply:
im a long way from gradle expert also so I just made minimum changes

Reply:
the def = above makes it different so that's why I was asking